omega/omega-context/src/main/java/org/apache/servicecomb/pack/omega/context/OmegaContextManager.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.pack.omega.context;
+
+public class OmegaContextManager {
```

Comment:
The OmegaContext should not be static, otherwise it could cause some trouble if there is more than one OmegaContext.

Reply:
we have used omegaContext as a global singletone bean before，it equal to static，threadlocal varible defined inside the omegaContext，it is thread safe